[DEV-904] refactor: SIEM Azion documentation for BB (#621)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/observe/siem/siem.mdx

permalink: /documentation/products/siem-azion/
---

> **Azion SIEM** allows you to expand visibility into accesses to your edge applications. By correlating data available on [Real-Time Metrics](/en/documentation/products/real-time-metrics/) and [Real-Time Events](/en/documentation/products/real-time-events/), you can amplify your analysis of security, access volume, health and degradation.

Reviewer: ```suggestion
**Azion SIEM** allows you to expand visibility into accesses to your edge applications. By correlating data available on [Real-Time Metrics](/en/documentation/products/real-time-metrics/) and [Real-Time Events](/en/documentation/products/real-time-events/), you can amplify your analysis of security, access volume, health, and degradation.
```

## Dashboards

> Each dashboard has a set of graphs and tables that allows you to monitor your applications according to different observability needs. By selecting different time ranges and using the available filters, you will be able to expand view of your applications, such as refining Edge Application settings, optimizing your Edge Firewall and WAF set rules, identifying _bots_ access, among others.

Reviewer: ```suggestion
Each dashboard has a set of graphs and tables that allows you to monitor your applications according to different observability needs. By selecting different time ranges and using the available filters, you'll be able to expand view of your applications, such as refining Edge Application settings, optimizing your Edge Firewall and WAF set rules, identifying *bots* access, among others.
```

## Alerts

> In addition to dashboards, SIEM also provides the creation and monitoring of alerts. Alerts can be viewed on the web interface and can also be sent by email, generating notifications at the beginning and end of the event.

Reviewer: ```suggestion
SIEM also provides the creation and monitoring of alerts. Alerts can be viewed on the web interface and can also be sent by email, generating notifications at the beginning and end of the event.
```

> Alerts can be configured considering a time range, a query using one or more items available in [Real-Time Events](/pt-br/documentacao/produtos/real-time-events/) and an expected threshold, such as the minimum, maximum value, average or sum of the collected values.

Reviewer: ```suggestion
Alerts can be configured considering a time range, a query using one or more items available in [Real-Time Events](/en/documentation/products/observe/real-time-events/) and an expected threshold, such as the minimum, maximum value, average, or sum of the collected values.
```

> Examples of alerts that can be created:

Reviewer: ```suggestion
Some examples of alerts that can be created:
```

> - Considering the last 5 minutes, generate an alert when the volume of requests per minute exceeds 1000 requests to any of the applications.

Reviewer: ```suggestion
- Considering the last 5 minutes, generate an alert when the volume of requests per minute exceeds 1,000 requests to any of the applications.
```
